.travis.yml

```diff
@@ -5,7 +5,6 @@ sudo: false
 dist: trusty
 rvm:
   - 2.2
-  - 2.3
```

Comment:
what about ruby 2.2 ?

Reply:
We're leaving 2.2 support in because it's still widely used; Ruby 2.3 has issues with bundler globally at the moment so I've dropped it here to allow this to pass.